Tests/ApolloCodegenTests/ApolloCodegenTests.swift

```diff
@@ -862,7 +862,7 @@ class ApolloCodegenTests: XCTestCase {
       input: .init(schemaPath: schemaPath, operationSearchPaths: [operationsPath]),
       output: .init(
         schemaTypes: .init(path: directoryURL.path,
-                           moduleType: .swiftPackageManager),
+                           moduleType: .swiftPackageManager()),
```

Comment:
Oh, looks like this is going to be a slight breaking change for existing users... Even with a default value, you have to add the `()`. Not ideal, but if it has to be, then we can live with this.

I don't think Swift will let overload enum case names with different associated values, but can you try it just in case? If we could have the old `swiftPackageManager` case (and mark it deprecated), along with the new `swiftPackageManager(version:) case, that would be better for backwards compatibility.

Otherwise, the only way to make this backwards compatible I can think of would be to deprecate the old case and create the new case with a different name. Something like `swiftPackage(version:)` might make sense? What do you two think?

Reply:
I can try the overload, I'm not against just deprecating the existing case and adding a new one, wouldn't take much to do and like you said avoids the minor breaking change.

Reply:
Confirmed the overload doesn't work, going to go the deprecation route with new case.